src/framework/cfgTools.ml

```diff
@@ -246,7 +246,7 @@ let createCFG (file: file) =
           let newst = mkStmt (Return (None, fd_loc)) in
           let start_id = 10_000_000_000 in (* TODO get max_sid? *)
           let sid = Hashtbl.hash fd_loc in (* Need pure sid instead of Cil.new_sid for incremental, similar to vid in Goblintutil.create_var. We only add one return stmt per loop, so the location hash should be unique. *)
-          newst.sid <- if sid < start_id then sid + start_id else sid;
+          newst.sid <- sid + start_id;
```

Comment:
Is this change required?

I'm not sure if it still guarantees that there are no `sid` smalller than `10_000_000_000` generated here? What if the hash already is large and then overflows when adding `10_000_000_000` and ends up in the territory where CILs normal ids are?

Reply:
Good point, I changed it back with efbe9e5ef0acc359cc5a03c9e5a0cb615970007a.